### PR TITLE
[L2B-5587] Add persistance for verifier status

### PR DESCRIPTION
**Pull request by maciekop-l2b**
Resolves L2B-5587

### CODE REVIEW

File: packages/backend/src/peripherals/database/types.ts [hunk 1]
```diff
@@ -261,12 +268,14 @@ declare module 'knex/types/tables' {
     finality: FinalityRow
     tracked_txs_configs: TrackedTxsConfigRow
     l2_costs: L2CostsRow
+    l2_costs_prices: L2CostsPricesRow
```
Comment (antooni) [inline packages/backend/src/peripherals/database/types.ts:271]: why this change?

**Review by michalsidzej — APPROVED** [review]
LGTM

File: packages/backend/src/peripherals/database/types.ts (hunk 1 above)
Comment (maciekop-l2b) [reply packages/backend/src/peripherals/database/types.ts:271]: I missed this before when the the table was being added

File: packages/backend/src/modules/verifiers/repositories/VerifierStatusRepository.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { Logger } from '@l2beat/backend-tools'
+import { ChainId, UnixTime } from '@l2beat/shared-pure'
+import { VerifierStatusRow } from 'knex/types/tables'
```
Comment (antooni) [inline packages/backend/src/modules/verifiers/repositories/VerifierStatusRepository.ts:3]: IMO it is easier to read if this type is also in the Repository .ts file

Comment (maciekop-l2b) [reply packages/backend/src/modules/verifiers/repositories/VerifierStatusRepository.ts:3]: On the other hand Row types are heavily coupled with knex table definitions so I would prefer to keep it here.  What's more I would like to prevent referencing modules here as it creates circular dependency: modules => peripherals => modules
